dotCMS/src/integration-test/java/com/dotcms/enterprise/license/LicenseManagerTest.java

```diff
@@ -28,7 +28,7 @@ public static void prepare() throws Exception {
     public void test_checkServerDuplicity_community() throws Exception {
         try {
             LicenseManager.getInstance().freeLicenseOnRepo();
-            assertEquals(LicenseLevel.COMMUNITY.level, LicenseManager.getInstance().license.level);
+            //assertEquals(LicenseLevel.COMMUNITY.level, LicenseManager.getInstance().license.level);
```

Comment:
why is this commented? Are you sure the license used is community?

Reply:
I'm rolling that back. not my changes. Probably came from the focal point branch I merged with